src/Workspaces/Core/Portable/Workspace/Solution/SolutionState.CompilationAndGeneratorDriverTranslationAction.cs

```diff
@@ -27,6 +27,15 @@ public virtual Task<Compilation> TransformCompilationAsync(Compilation oldCompil
             /// side effects. This opts those out of operating on ones with generated documents where there would be side effects.
             /// </remarks>
             public abstract bool CanUpdateCompilationWithStaleGeneratedTreesIfGeneratorsGiveSameOutput { get; }
+
+            /// <summary>
+            /// When changes are made to a solution, we make a list of translation actions. If multiple similar changes happen in rapid
+            /// succession, we may be able to merge them without holding onto intermediate state.
+            /// </summary>
+            /// <param name="priorAction">The action prior to this one. May be a different type.</param>
+            /// <returns>A non-null <see cref="CompilationAndGeneratorDriverTranslationAction" /> if we could create a merged one, null otherwise.</returns>
```

Comment:
```suggestion
            /// <returns>A <see cref="CompilationAndGeneratorDriverTranslationAction"/> if we could create a merged one; otherwise, <see langword="null"/> if the actions could not be merged.</returns>
```

Reply:
Never a fan of duplicating the condition like this.